subjects/mini-framework/README.md

```diff
@@ -0,0 +1,105 @@
+## mini-framework
+
+Now that you have already used a framework of your choice, you must now implement some features on a framework of your own. That's right, you are going to create a framework.
+
+Be aware that a framework is different from a library. When you call a method from a library, you are in control. But with a framework, the control is inverted: the framework calls you.
+
+### Objectives
+
+Your framework should implement:
+
+- Abstracting DOM
+- Easier Routing
```

Comment:
Not very important but I don't like `Easier` as a way to describe the work needed to be done for the routing... but I'm not sure about what would be better so haha !

Maybe: `Routing System` ?

Reply:
Yeah honestly, routing system seems better ahah